README.md

```diff
@@ -1 +1,70 @@
-# Awesome-book
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Awesome-book
+
+> Building A Library-like Books Management App
+
+![screenshot](./assets/images/app_screenshot.png)
+
+This is a Todo-like book management app to showcase the skills we have acquired in JavaScript Object Oriented Programmin.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+
+[Live Demo Link](https://naughty-jackson-467756.netlify.app/)
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+Clone the repo with `git clone https://github.com/HENRYKC24/my-portfolio.git`
+
+Run `npm install` from the command line
+
+Run `npx hint .` to test for html linting check
+
+Run `npx stylelint "**/*.{css,scss}"` to test for css linting check 
+
+
+The app is deployed using GitHub pages. You can view it using the 'live demo' above.
+
+
+
+## Author
+👤 **Ali Abbani**
+👤 **Henry Kc**
+## Henry's Profile
+- GitHub: [@githubhandle](https://github.com/henrykc24)
+- Twitter: [@twitterhandle](https://twitter.com/henrykc24)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/henry-kc)
+
+## Ali's Profile
+- GitHub: [@githubhandle](https://github.com/aliabbani)
+- Twitter: [@twitterhandle](https://twitter.com/aliabbani)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ali-abbani-8b6246150/)
```

Comment:
- [ ] There are 2 authors of this project. Please change the subheading `Author` to `Authors`, and  _Ali Abbani's_ name should appear directly under Ali's profile/ details